.buildkite/build_apk.sh

```diff
@@ -1,26 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
Stuff here is broken and the idea that the logic should be in the buildkite script is flawed: It should be in the Docker container such that it's runable + dev'able locally.

Choosing to delete instead of maintain for now, we can always rediscover this in git history.

Reply:
Would love to learn more about this!